src/components/PostSummary.vue

```diff
@@ -135,7 +135,7 @@ export default {
 </script>
 
 <style scoped lang="stylus">
-@import './../css/variables'
+/* @import './../css/variables' */
```

Comment:
Wondering what's going on here, is it just not being used or is it causing problems?

Reply:
Ah, my bad, i'll revert it now. It was my first stab at guessing what was wrong 😅